src/app/pages/PhotoGalleryPage/index.jsx

```diff
@@ -108,6 +109,7 @@ const PhotoGalleryPage = ({ pageData }) => {
         aboutTags={aboutTags}
       />
       <ATIAnalytics data={pageData} />
+      <ChartbeatAnalytics data={pageData} />
```

Comment:
Just looking at the page tests for STY pages. We seem to be mocking [chartbeat](https://github.com/bbc/simorgh/blob/3eed623efc9ec02764960567aebf6848d572055b/src/app/pages/StoryPage/index.test.jsx#L39), do we want to do the same for the PGL page test?

Reply:
good idea. Just doing a bit of a reactor on the tests, will include it in that update